### ISSUE TITLE
Still NumberFormatException in updateIntellij 

### ISSUE THREAD

**Issue by exoego** [JetBrains/sbt-idea-plugin#40]
#37 is resolved in https://github.com/JetBrains/sbt-idea-plugin/commit/b1e825d00f1dde9110f799e3d3b04a93a78c913e,
but now the below error shows up after rebooting sbt after successful installation.

```
(ThisBuild / updateIntellij) java.lang.NumberFormatException: For input string: "192-2147483647"
```

**Comment by mutcianm** [comment]
This seems to be unrelated to previous bug. 
Apparently some plugin authors are not following IDEA's versioning convention(

**Comment by mutcianm** [comment]
No, thanks, it's fine. I'll just implement a more generic version comparison.
Easier than reaching out to everyone who has malformed version string.
